cypress/plugins/index.js

```diff
@@ -253,6 +242,33 @@ function deleteCollectionRecursively(collection) {
   });
 }
 
+/**
+ * Stores necessary data for tests to consume from prod Firebase. Called
+ * once at the start of all tests. We
```

Comment:
nit: Initially read this as all the tests in a single test file

Reply:
That's correct, clarified